single-user-ai4pp/Dockerfile

```diff
@@ -64,10 +67,10 @@ RUN pip install --no-cache-dir \
 	jupyter-tensorboard-proxy \
 	mlflow-server-proxy \
 	protobuf==4.21.9 \
-	torch==1.12.1+cu113 \
-	torchvision==0.13.1+cu113 \
-	torchaudio==0.12.1 \
-	--extra-index-url https://download.pytorch.org/whl/cu113 	
+	# torch==1.12.1+cu113 \
+	# torchvision==0.13.1+cu113 \
+	# torchaudio==0.12.1 \
+	--extra-index-url https://download.pytorch.org/whl/cu113
```

Comment:
this line is because the torch above but now is commented, not sure if it makes sense to keep